romana-install/romana-setup

```diff
@@ -17,14 +17,15 @@
 
 usage() {
 	echo "romana-setup: executes the ansible playbooks for installing romana"
-	echo "Usage: romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [action]"
-	echo "       romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] <action> [ansible-options]"
+	echo "Usage: romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [-c number] [action]"
+	echo "       romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [-c number] <action> [ansible-options]"
 	echo "       romana-setup <h|--help>"
 	echo ""
 	echo "Cluster Name:  user-defined stack name (default: $USER)"
 	echo "Platforms:     aws (default), vagrant, static"
 	echo "Distro:        ubuntu (default), centos"
 	echo "Stack Types:   devstack (default), kubernetes"
+	echo "Number:        number of compute nodes"
```

Comment:
I suppose the intention of it is number of compute-only / minion-only nodes.

Cluster-type independent.. now I'm tempted to rename other things based on cluster type ;-)
(Eg: create host names with `-controller`, `-compute01` etc for devstack/openstack and `-master`, `-minion01`, etc) for kubernetes.)
Maybe later for that. I'll update the wording to match its intention of being additional nodes.